src/index.js

```diff
@@ -80,14 +80,30 @@ const mouse = (state = {}, action) => {
   }
 }
 
+const frequency = (state = 0, action) => {
+  switch (action.type) {
+    case 'SET_FREQUENCY_BY_FIELD':
+      return action.frequency
+    case 'SET_FREQUENCY_BY_SLIDER':
+      return logarithmicScaleToFrequency(action.power)
```

Comment:
at some point we should move this and `frequencyToLogarithmicScale` to their own module...`math` or something.

Reply:
yeah I figured I would hold off on that for a bit cause I'm not sure what math we're going to need/not need in the end. though I suppose it wouldn't hurt to just start now